.vscode/tasks.json

```diff
@@ -0,0 +1,20 @@
+// See https://go.microsoft.com/fwlink/?LinkId=733558
+// for the documentation about the tasks.json format
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "npm",
```

Comment:
Does this work without issues in combination with pnpm?

Reply:
I think it would work in this case because this specific task just calls `tsc -watch -p ./`, but I have changed it to a shell task that calls pnpm explicity.